src/components/Video/Video.spec.tsx

```diff
@@ -27,8 +27,11 @@ test('Renders an iframe with the correct src and title', () => {
 
 test('Includes necessary attributes in the iframe element', () => {
   render(<Video {...testProps} />)
-
+  const videoCoverImage = screen.getByRole('img', { name: testProps.title })
+  expect(videoCoverImage).toBeVisible()
+  videoCoverImage.click()
   const videoElement = screen.getByTitle(testProps.title)
+  expect(videoElement).toBeVisible()
```

Comment:
So I would add more thorough checks as we don't know what the video element is, just that it exists.  Add some attribute checks just to be sure